dist/index.js

```diff
@@ -0,0 +1,86 @@
+"use strict";
```

Comment:
We can of course gitignore `dist/` folder. I just left it there to be able to install this package from Github.